CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### App Center Crashes
 
 * **[Fix]** Remove the multiple attachments warning as that is now supported by the portal.
+* **[Fix]** Fix remove minidump files when the sending crash report was discarded.
```

Comment:
I see that processing rules are changed, so the main fix should be about processing and sending the crashes but not files clean up, right?

Reply:
I think it can be written as two separate items. Something about fixing the minidump filter should also be added.

Reply:
I add one more line to address changes in this PR more correctly.